tests/main/views/test_marketplace.py

```diff
@@ -551,10 +586,6 @@ def test_supplier_start_application(self):
 
     def test_supplier_applied_view_application_for_live_opportunity(self):
         self.login_as_supplier()
-        # mocking that we have applied
```

Comment:
These tests should include some set up that sets the supplier who is logged in as a supplier who has applied successfully (i.e. a submitted brief response)

I think these might need adding back in and changing so that they include a response for the supplier ID e.g.
`self._data_api_client.find_brief_responses.return_value = {
            "briefResponses": [{"supplierId": supplier id goes here}],
}
`

I think this is due to incorrect use of `supplier_id` rather than `supplierId` in your view which is searching for one response for a supplier.